lib/panoptes-subject-uploader.iced

```diff
@@ -30,13 +30,20 @@ findImagesFiles = (searchDir, metadata) ->
   imageFiles
 
 findImagesURLs = (metadata) ->
-  imageURLs = []
+  httpsImageURLs = []
   for key, value of metadata
+
     httpsRegexPattern = /^(https):\/\//i
     fileRegexPattern = /([^\/]+.(?:jpg|jpeg|gif|png|svg|mp4|txt))$/i
-    if httpsRegexPattern.test(value) and fileRegexPattern.test(value)
-      imageURLs.push value
-  imageURLs
+
+    if fileRegexPattern.test(value)
```

Comment:
Update from our discussion in slack: 
All links included without media file extensions (i.e., jpg, jpeg, gif, png, svg, mp4, txt) will be included on the subject as metadata. I will add notes to the documentation for clarity.